javascript/javascript_sdk/src/message_receiver.js

```diff
@@ -1,57 +1,57 @@
 'use strict';
 
-const zmq = require('zeromq');
-
+const {OutputServiceClient} = require('./../js_proto/output_grpc_web_pb.js');
 const parsing = require('./parse_output');
 
 
 class MessageReceiver {
   constructor() {
-    this._socket = new zmq.Subscriber;
+    //this._socket = new zmq.Subscriber;
     this._currentConnectedAddress = null;
   }
 
   connect(address = 'localhost') {
-    this._currentConnectedAddress = `tcp://${address}:5050`;
+    this._currentConnectedAddress = `http://${address}:8080`;
     console.log(`Connecting to ${this._currentConnectedAddress}`);
-    this._socket.connect(this._currentConnectedAddress);
-    this._socket.subscribe();
+    this._outputService = new OutputServiceClient(_currentConnectedAddress);
+    var request = new OutputRequest();
+    request.setMessage('Hello World!');
```

Comment:
Is this meant to be here or just for testing?

Reply:
Sorry, all the code under javascript will be retreat in the other PR. Sorry for the inconvenience.